specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/privateLinkService.json

```diff
@@ -740,13 +754,13 @@
           "$ref": "./virtualNetwork.json#/definitions/Subnet",
           "description": "The reference of the subnet resource."
         },
-        "publicIPAddress": {
```

Comment:
removing an existing property (stable/GA api version) is considered as a breaking change. @bowen0106 - why do you need to remove this property. Anton Evseev tells me that this API is behind a feature flag and all the customers using it have been notified of this change. Can you please confirm?

Reply:
Yes. It’s confirmed. This feature currently is testing with limited access guarded by feature flag from Subscription Level.

Reply:
Currently there is no external customer using this feature.